src/app/core/services/attachment/attachment.service.ts

```diff
@@ -61,7 +61,7 @@ export class AttachmentService {
         this.actionSheetController.dismiss('removeCurrentPhoto');
         this.presentToast(this.texts["REMOVE_CURRENT_PHOTO"], "success");
     }
-    async selectImage(type) {
+    async selectImage(profileImageData) {
```

Comment:
same here, change the name after passing the type instead of this

Reply:
Remove current photo option check whole profile image data is needed